infrastructure/.env.example

```
@@ -3,6 +3,8 @@ LIVEKIT_API_SECRET=devsecret
 LIVEKIT_WS_URL=ws://localhost:7880
 AUTH_BIND_ADDR=0.0.0.0:8787
 AUTH_DATABASE_URL=sqlite:///app/data/auth.db?mode=rwc
+AUTH_COOKIE_SECRET=dev-cookie-secret-change-me
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_COOKIE_SECRET key should go before the AUTH_DATABASE_URL key

(UnorderedKey)